src/tools/ceph-dencoder/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(dencoder_srcs
   osd_types.cc
   rbd_types.cc
   rgw_types.cc
+  ../../librbd/cache/pwl/Types.cc
```

Comment:
Nit: please keep all the RBD types in `rbd_types` static library

Reply:
sorry, misunderstood your earlier statement. Fixed it.